referencemodels/src/test/java/org/openehr/referencemodels/RMComparedWithBmmTest.java

```diff
@@ -140,6 +140,10 @@ public void compareBmmWithRM() {
         knownDifferences.add(new ModelDifference(ModelDifferenceType.PROPERTY_MISSING_IN_BMM, "", "IMPORTED_VERSION", "canonical_form"));
         knownDifferences.add(new ModelDifference(ModelDifferenceType.PROPERTY_MISSING_IN_BMM, "", "OBJECT_VERSION_ID", "extension"));
         knownDifferences.add(new ModelDifference(ModelDifferenceType.PROPERTY_MISSING_IN_BMM, "", "OBJECT_VERSION_ID", "root"));
+        knownDifferences.add(new ModelDifference(ModelDifferenceType.PROPERTY_MISSING_IN_BMM, "", "OBJECT_VERSION_ID", "object_id"));
```

Comment:
this does mean these methods also get added to the model, and are usable in Archetypes (if the BMM is not used as meta model) and path lookups. Is that desired behaviour?

Reply:
I don't know, sorry. I was just adding lines here until the test was successful.